Build errors in src/tree/integration-account

````
src/tree/integration-account/IntegrationAccountMapsTreeItem.ts:59:30
- error TS6133: 'node' is declared but its value is never read.

59     public async createChild(node: IAzureNode, showCreatingNode: (label: string) => void, userOptions?: any): Promise<IAzureTreeItem> {
                                ~~~~

src/tree/integration-account/IntegrationAccountMapsTreeItem.ts:59:91
- error TS6133: 'userOptions' is declared but its value is never read.

59     public async createChild(node: IAzureNode, showCreatingNode: (label: string) => void, userOptions?: any): Promise<IAzureTreeItem> {
                                                                                             ~~~~~~~~~~~
````

--- a/src/tree/integration-account/IntegrationAccountMapsTreeItem.ts
+++ b/src/tree/integration-account/IntegrationAccountMapsTreeItem.ts
@@ -56,30 +56,30 @@ export class IntegrationAccountMapsTreeItem implements IAzureParentTreeItem {
         return integrationAccountMaps.map((map: IntegrationAccountMap) => new IntegrationAccountMapTreeItem(this.client, map));
     }
 
-    public async createChild(node: IAzureNode, showCreatingNode: (label: string) => void, userOptions?: any): Promise<IAzureTreeItem> {
+    public async createChild(_: IAzureNode, showCreatingNode: (label: string) => void): Promise<IAzureTreeItem> {
         const mapTypes = Object.keys(MapType).map((k) => MapType[k as any]);
         const mapType = await vscode.window.showQuickPick(mapTypes);
 
         if (mapType) {
             const mapName = await vscode.window.showInputBox(
-            {
-                prompt: "Enter the name of your new Map",
-                validateInput: async (value: string): Promise<string | null> => {
-                    const existingMaps = await this.getAllMaps();
-                    if (existingMaps && existingMaps.find((map) => map.name === value)) {
-                        return localize("azIntegrationAccounts.nameAlreadyInUse", "Name already in use");
-                    }
+                {
+                    prompt: "Enter the name of your new Map",
+                    validateInput: async (value: string): Promise<string | null> => {
+                        const existingMaps = await this.getAllMaps();
+                        if (existingMaps && existingMaps.find((map) => map.name === value)) {
+                            return localize("azIntegrationAccounts.nameAlreadyInUse", "Name already in use");
+                        }
 
-                    return null;
-                }
-            });
+                        return null;
+                    }
+                });
 
             if (mapName) {
                 showCreatingNode(mapName);
                 const newMap: IntegrationAccountMap = await this.client.maps.createOrUpdate(this.resourceGroupName,
-                                                                    this.integrationAccountName,
-                                                                    mapName,
-                                                                    await createNewMap(mapName, (MapType as any)[mapType]));
+                    this.integrationAccountName,
+                    mapName,
+                    await createNewMap(mapName, (MapType as any)[mapType]));
 
                 return new IntegrationAccountMapTreeItem(this.client, newMap);
             }

--- a/src/tree/integration-account/IntegrationAccountSchemasTreeItem.ts
+++ b/src/tree/integration-account/IntegrationAccountSchemasTreeItem.ts
@@ -56,26 +56,26 @@ export class IntegrationAccountSchemasTreeItem implements IAzureParentTreeItem {
         return integrationAccountSchemas.map((schema: IntegrationAccountSchema) => new IntegrationAccountSchemaTreeItem(this.client, schema));
     }
 
-    public async createChild(node: IAzureNode, showCreatingNode: (label: string) => void, userOptions?: any): Promise<IAzureTreeItem> {
+    public async createChild(_: IAzureNode, showCreatingNode: (label: string) => void): Promise<IAzureTreeItem> {
         const schemaName = await vscode.window.showInputBox(
-        {
-            prompt: "Enter the name of your new Schema",
-            validateInput: async (value: string): Promise<string | null> => {
-                const existingSchemas = await this.getAllSchemas();
-                if (existingSchemas && existingSchemas.find((schema) => schema.name === value)) {
-                    return localize("azIntegrationAccounts.nameAlreadyInUse", "Name already in use");
-                }
+            {
+                prompt: "Enter the name of your new Schema",
+                validateInput: async (value: string): Promise<string | null> => {
+                    const existingSchemas = await this.getAllSchemas();
+                    if (existingSchemas && existingSchemas.find((schema) => schema.name === value)) {
+                        return localize("azIntegrationAccounts.nameAlreadyInUse", "Name already in use");
+                    }
 
-                return null;
-            }
-        });
+                    return null;
+                }
+            });
 
         if (schemaName) {
             showCreatingNode(schemaName);
             const newSchema: IntegrationAccountSchema = await this.client.schemas.createOrUpdate(this.resourceGroupName,
-                                                                this.integrationAccountName,
-                                                                schemaName,
-                                                                await createNewSchema(schemaName));
+                this.integrationAccountName,
+                schemaName,
+                await createNewSchema(schemaName));
 
             return new IntegrationAccountSchemaTreeItem(this.client, newSchema);
         }

